packages/react-search-ui/src/__tests__/SearchProvider.test.js

```diff
@@ -1,3 +1,28 @@
-it("needs tests", () => {
-  expect(true).toBe(true);
+import React from "react";
+import { mount } from "enzyme";
+
+import { SearchProvider, SearchConsumer } from "../..";
+
+describe("SearchProvider", () => {
+  it("exposes state and actions to components", () => {
```

Comment:
Same question here. Should we test that we expose actions or is testing that we expose state is enough?